### PR TITLE
[feature](merge-cloud) Add show data warehouse stmt

### PR DESCRIPTION
**Pull request by swjtu-zhanglei**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowDataStmt.java [hunk 1]
```diff
@@ -101,15 +108,78 @@ public class ShowDataStmt extends ShowStmt {
...
         this.tableName = tableName;
         this.totalRows = Lists.newArrayList();
         this.orderByElements = orderByElements;
+        this.properties = properties;
     }
 
     @Override
     public void analyze(Analyzer analyzer) throws UserException {
         super.analyze(analyzer);
+        if (properties != null) {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowDataStmt.java:126]: consider using a separate mothod

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowDataStmt.java (hunk 1 above)
Comment (SWJTU-ZhangLei) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowDataStmt.java]: > consider using a separate mothod

resolved

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**